tests/workflow_contracts/makefile_cargo_resolution_test.py

```
@@ -0,0 +1,93 @@
+    """Inputs and expected source for one Cargo resolution scenario."""
+
+    cargo_override: str
+    has_path_cargo: bool
+    has_home_cargo: bool
+    expected_location: str
+
+
+@pytest.mark.parametrize(
+    "case",
+    [
+        CargoResolutionCase("", True, False, "path"),
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)